cmd/skaffold/app/cmd/build.go

```diff
@@ -51,6 +51,7 @@ func NewCmdBuild() *cobra.Command {
 		WithCommonFlags().
 		WithFlags(func(f *pflag.FlagSet) {
 			f.StringSliceVarP(&opts.TargetImages, "build-image", "b", nil, "Choose which artifacts to build. Artifacts with image names that contain the expression will be built only. Default is to build sources for all artifacts")
+			f.StringVarP(&opts.CustomTag, "tag", "t", "", "The optional custom tag to use for images which overrides the current Tagger configuration")
```

Comment:
This should go in https://github.com/GoogleContainerTools/skaffold/blob/master/cmd/skaffold/app/cmd/flags.go and should be registered for "run", "build", "dev" and "debug"

I think this cycle we are working on Multiple taggers.
Its strange, we only had this for Run and not for dev and debug which also runs skaffold build.

I am curious are you planning to use this in your CI/CD pipeline?

Reply:
Read the linked issue.

Reply:
can move this to flags.go and register it for `build` and `run` only for now.